skimage/measure/profile.py

```diff
@@ -44,11 +46,18 @@ def profile_line(img, src, dst, linewidth=1,
            [0, 0, 0, 0, 0, 0]])
     >>> profile_line(img, (2, 1), (2, 4))
     array([ 1.,  1.,  2.,  2.])
+    >>> profile_line(img, (1, 0), (1, 6), cval=4)
+    array([ 1.,  1.,  1.,  2.,  2.,  2.,  4.])
 
     Notes
     -----
     The destination point is included in the profile, in contrast to
     standard numpy indexing.
+
+    For example: 
+        >>> img = np.array([1,1,2,2])
+        >>> profile_line(img, (0, 0), (0, 4)) # The final point is out of bounds
+        >>> profile_line(img, (0, 0), (0, 3)) # This accesses the full first row
```

Comment:
Thanks! All of this should go in the Examples section. Note that doctest allows you to mix code and prose, just separate by blank lines. See: https://docs.python.org/3.5/library/doctest.html

You should add the output and make sure it's correct by running it. Remember that even a single stray character can cause a doctest to fail!